### PR TITLE
 OWLS-75359, OWLS-75638, added test to cover endtoend monitoring exporter sample

### PR DESCRIPTION
**Pull request by marinakog**

### CODE REVIEW

**Comment by marinakog** [comment]
jenkin passed, ready for review
http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/1922/

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java [hunk 1]
```diff
@@ -117,7 +119,7 @@ public static void staticUnPrepare() throws Exception {
       if (operator != null) {
         operator.destroy();
       }
-      deletePrometheusGrafana();
+      // deletePrometheusGrafana();
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java]: No need to keep commented out code

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java [hunk 2]
```diff
@@ -776,6 +793,79 @@ public void test18_ChangeConfigEmptyPass() throws Exception {
     logger.info("SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Test End to End example from MonitoringExporter github project
+   *
+   * @throws Exception
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java]: bad javadoc.  Need a description after Exception (even if very short)

**Comment by vanajamukkara** [comment]
@marinakog please update USECASES.MD

**Comment by marinakog** [comment]
> I see hard-coded password welcome1. Can you use the variable available in base test

replaced with values from BaseTest

**Pull request merged**